9c-main/chart/templates/remote-action-evaluator-headless.yaml

```diff
@@ -365,6 +377,10 @@ data:
             {
               "path": "https://9c-dx.s3.ap-northeast-2.amazonaws.com/lib9c-stateservices/170da2660770f6dec9911ba174618a8c5b154dca/linux-x64.zip",
               "port": 11111
+            },
+            {
+              "path": "https://9c-dx.s3.ap-northeast-2.amazonaws.com/lib9c-stateservices/1a2887911c7ba4492fa9eb99564a58f0f461a42b/linux-x64.zip",
```

Comment:
@riemannulus It is my mistake, it should use arm64 image. The live manifest may use linux-arm64